controllers/users.controller.js

```diff
@@ -13,6 +13,21 @@ const create = async (req, res) => {
   }
 };
 
+const update = async (req, res, next) => {
+  try {
+      const id = req.params.id;
+      const updates = req.body
```

Comment:
You'd rather do
```js
const { params, body } = req;
const { id } = params;

// pass body as it is to your db model instead of assigning it to a new variable, that's useless in this case
```